Source/BTNavigationDropdownMenu.swift

```diff
@@ -222,7 +226,7 @@ public class BTNavigationDropdownMenu: UIView {
         self.backgroundView.alpha = self.configuration.maskBackgroundOpacity
         
         // Animation
-        UIView.animateWithDuration(self.configuration.animationDuration, delay: 0.15, options: UIViewAnimationOptions.TransitionNone, animations: {
```

Comment:
Why do you have this delay?

I think delay should be zero because animation is faster.